src/org/sosy_lab/java_smt/solvers/cvc5/CVC5StringFormulaManager.java

```diff
@@ -27,7 +27,8 @@ class CVC5StringFormulaManager extends AbstractStringFormulaManager<Term, Sort,
 
   @Override
   protected Term makeStringImpl(String pValue) {
-    return solver.mkString(escapeUnicodeForSmtlib(pValue), true);
+    String str = escapeUnicodeForSmtlib(unescapeUnicodeForSmtlib(pValue));
```

Comment:
I would assume that `escape` and `unescape` are exactly reversed operations. This line is then a no-op and not required.

Reply:
fixed